.gitbook/0-about-panacea/1-data-exchange-protocol/0-what-is-dep.md

```diff
@@ -0,0 +1,37 @@
+# What is DEP?
+
+## Overview
+
+DEP is a protocol for delivering sensitive data in a decentralized way while considering the privacy of the data.
+In order to deliver sensitive data (e.g., medical data) in a decentralized environment, 
+data encryption is essential, and as a trade-off, it makes difficult to verify whether the data is valid or not.
+DEP goes beyond these limitations and aims for a protocol that allows sensitive data to be delivered securely and validation of the data.
+
+With DEP, data consumers can specify the type of data they want in a deal, 
+and data providers can be guaranteed that their data is delivered securely with data privacy. 
+With DEP, data providers can take ownership of their data and receive the proper compensation,
+while data consumers can gain access to sensitive data that would otherwise be difficult to collect 
+and utilize for various research or business purposes.
+
+## DEP Component
+The DEP ecosystem consists of consumer, provider, Panacea blockchain, and oracle as components.
```

Comment:
Did the component needs to change `Panacea blockchain validator`? Or the just leave it?
I'm not sure.

Reply:
That seems clearer, thanks.